src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -599,11 +624,15 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
             enddo
          enddo
          !$omp end do
+         !$omp end parallel
       endif
 
       nEdges = nEdgesArray( 3 )
 
-      !$omp do schedule(runtime)
+      !!$omp parallel
+      !!$omp do schedule(runtime) &
+      !!$omp private(iEdge, cell1, cell2, k, BruntVaisalaFreqTopEdge, tridiagB, tridiagC, &
+      !!$omp         rightHandSide, tridiagA, N)
```

Comment:
This loop has the OpenMP directives commented out since there is a race condition that is yet to be resolved.  This will be resolved in a future PR (the work in this PR need to be merged for future work to build upon).